daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -210,7 +214,25 @@ object SValue {
   // with SValue and we can remove one layer of indirection.
   sealed trait SPrimLit extends SValue with Equals
   final case class SInt64(value: Long) extends SPrimLit
+  // TODO https://github.com/digital-asset/daml/issues/8719
+  //  try to factorize SNumeric and SBigNumeric
+  //  not it seems that scale is relevant in SNumeric but lost in SBigNumeric
```

Comment:
typo? Not quite sure what the `not` is supposed to mean here.

Reply:
"note".